### PR TITLE
[March 24] Feature/gm reorganisation March 2022

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard24**
- Generated new maps for the gm reorganization of March 2022. (Weesp -> Amsterdam)
- Redirect for old gm
- Docs update

GM map update:

Screenshot 2022-03-15 at 16 20 16

Screenshot 2022-03-15 at 16 20 34

---

VR map update:

Screenshot 2022-03-15 at 16 42 31

Screenshot 2022-03-15 at 16 42 43

### CODE REVIEW

**Comment by VWSCoronaDashboard24** [comment]
JSON validation will fail at this point because Weesp is still in the dataset.

File: packages/app/next.config.js [hunk 1]
```diff
@@ -172,20 +176,20 @@ const nextConfig = {
...
-      type: 'asset',
-      use: 'svgo-loader',
-      generator: {
-        filename: 'static/image/[name].[hash][ext]',
-      },
-      parser: {
-        dataUrlCondition: {
-          maxSize: 2 * 1024, // only inline SVG's < 2kB
+    (config.optimization.chunkIds = 'named'),
+      config.module.rules.push({
```
Comment (VWSCoronaDashboard18) [inline packages/app/next.config.js:180]: Why this code formatting indent? They are siblings right, not nested?

File: packages/app/next.config.js [hunk 2]
```diff
@@ -172,20 +176,20 @@ const nextConfig = {
...
+        test: /\.svg$/,
+        type: 'asset',
+        use: 'svgo-loader',
+        generator: {
+          filename: 'static/image/[name].[hash][ext]',
```
Comment (VWSCoronaDashboard18) [inline packages/app/next.config.js:185]:
```suggestion
    config.module.rules.push({
      test: /\.svg$/,
      type: 'asset',
      use: 'svgo-loader',
      generator: {
        filename: 'static/image/[name].[hash][ext]',
```

File: packages/app/next.config.js [hunk 3]
```diff
@@ -172,20 +176,20 @@ const nextConfig = {
...
         },
-      },
-    });
+        parser: {
+          dataUrlCondition: {
+            maxSize: 2 * 1024, // only inline SVG's < 2kB
+          },
+        },
+      });
```
Comment (VWSCoronaDashboard18) [inline packages/app/next.config.js:192]:
```suggestion
      parser: {
        dataUrlCondition: {
          maxSize: 2 * 1024, // only inline SVG's < 2kB
        },
      },
    });
```

File: packages/app/next.config.js (hunk 1 above)
Comment (VWSCoronaDashboard24) [reply packages/app/next.config.js]: Fixed!

**Review by VWSCoronaDashboard19 — APPROVED** [review]
So far so good 👍

**Pull request merged**